lib/components/DisqusThread.js

```diff
@@ -97,17 +97,8 @@ module.exports = React.createClass({
 
   render() {
     return (
-      <div {...this.props}>
```

Comment:
It removes the warning reported in #25 React now throws a warning if attributes are added to a div that are not known html attributes. As far as I could tell they are not being used for anything?